docs/samples/calling/app.js

```diff
@@ -601,18 +610,17 @@ function clearMediaDeviceList() {
 }
 
 async function getMediaDevices() {
-  const cameras = await window.Media.Media.getCameras();
-
+  const cameras = await calling.Media.Media.getCameras();
```

Comment:
Try accessing media directly from the calling client, we might not need to re-export it from index.ts and populate it inside calling instance. callingClient.mediaEngine.Media should suffice. Try this and update the change if it works

Reply:
`callingClient.mediaEngine.Media` worked. Thanks